dbms/src/DataStreams/CreatingSetsBlockInputStream.cpp

```diff
@@ -291,4 +291,16 @@ void CreatingSetsBlockInputStream::createOne(SubqueryForSet & subquery)
     }
 }
 
+uint64_t CreatingSetsBlockInputStream::collectCPUTimeNsImpl(bool /*is_thread_runner*/)
```

Comment:
`CreatingSetsBlockInputStream` is executed in `MPPTask::runImpl`, and I think we don't collect cpu time for that thread, is it the expected behavior?

Reply:
Yes, I don't think it is necessary to distinguish what thread is, mpptaskImpl threads are treated equally.
Usually the last child is a UnionBlockInputStream which doesn't count its own cpu time but its children's.
IMO, It depends on what stream last child is